CryptX/ApplicationLifeCycle/SettingsManager.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  SettingsManager.swift
+//  CryptX
+//
+//  Created by Mehmet Barkın Mercan on 20.07.2024.
+//
+
+import Foundation
+import UIKit
+
+struct CryptoCoin: Codable {
```

Comment:
This struct is shared between SettingsManager and CoinManager. It should not be in the same swift file as one of them, it should be in a separate file to locate it easily.